client/src/components/Settings/SettingsUniquenessScoreBox/SettingsUniquenessScoreBox.tsx

```diff
@@ -87,11 +87,9 @@ const SettingsUniquenessScoreBox = (): ReactNode => {
         <div className={styles.buttonsWrapper}>
           <Button
             className={styles.button}
+            isHigh
```

Comment:
Any `isLoading` when the request is being processed?

Reply:
This is update for recalculation (box is under the modal). I will handle this initial loading in [this issue](https://linear.app/golemfoundation/issue/OCT-1737/fetch-delegation-addresses-from-be).